libs/cases/frontend/view/src/lib/components/create-case/create-case.component.html

```diff
@@ -7,51 +7,74 @@
             #caseForm="ngForm"
             (submit)="onSubmit(caseForm)"
         >
-
+            <!-- Facility Selection -->
             <ix-select
                 id="facilitySelection"
                 (valueChange)="onFacilityChange($event)"
                 (inputChange)="onFacilityInputChange($event)"
+                hide-list-header="true"
```

Comment:
If I see it correctly that hide-list-header removes the "Please select an option" showing at the top of the selection
![grafik](https://github.com/amosproj/amos2024ss01-xcelerator-demo-app/assets/166606862/375f850e-cd8d-4edd-9ca2-031deecee6bb)

If you don't like it we can definitely remove it but you should also do it for the other  2 \<ix-select\> then

Reply:
Ty for removing it